hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import json
+
+from hmalib.common import get_logger
+from hmalib.models import MatchMessage
+
+logger = get_logger(__name__)
+
+def lambda_handler(event, context):
+    """
+    The output SNS topics from matchers have SQS instances subscribed to them.
+    This indirection allows for fanout. Multiple SQS queues can be tagged to the
+    same SNS topic and SNS will guarantee delivery to the queues.
+
+    Architecture aside, here's what you need to know. The actions layer will be
+    listening on an SQS queue. For every record,
+    - it will construct the MatchMessage object and identify the actions it
+      needs to invoke.
+    - for now, rather than fanning out to individual specific lambdas, it will
+      call all specific functions serially!
```

Comment:
I can't tell if this is boilerplate/copypaste from an example or specific to actioner.

Reply:
I wanted it to be specific to the 'action_performer'. Can you tell me what feels vague?

Reply:
It's hard to put my finger on, but I think a description of what the module does (the actioner converts matches into results from actions) rather than how it does it (sns queues with subscriptions) might help.